client/styles/components/_help-modal.scss

```diff
@@ -0,0 +1,39 @@
+.help-modal {
+  @extend %modal;
+  padding: #{20 / $base-font-size}rem;
+  width: #{500 / $base-font-size}rem;
+}
+
+.help-modal__header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.help-modal__section {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: #{10 / $base-font-size}rem 0;
+
+  // Basic text styles for help body text
+  ul,
```

Comment:
I thought SASS would expand these rules to `.help-modal__section ul` to scope it to the `HelpModal` content? I wasn't sure we'd want to apply globally.

But I can totally move to `_bass.scss` if you prefer?